tests/functional/test-connection-ok/source/data/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "action": "testConnection",
+  "parameters": {
+    "db": {
+      "#connectionString": "%env(string:CONNECTION_STRING)%"
```

Comment:
tam se bude zadávat celý connectionString?

Reply:
Ano, je to z 2 dovodov:
1. Priamo Azure v UI aj vsetkych tutorialoch uvadza `connection string`, ... teda nechceme po uzivatelovi aby to rucne parsoval (aj ked v screenshote je to aj oddelene, ale na inych miestach UI je len connection string).
2. Extractor funguje aj pre `ComsosDB Table API` aj pre `Storage Table`, kedze maju rovnaky interface,  ... no format connection string sa moze lisit.

![screenshot-portal azure com-2020 11 30-{time)](https://user-images.githubusercontent.com/19371734/100602284-fbf52e80-3303-11eb-8b41-5b9ff522b0f2.png)